portfolio/src/main/webapp/index.html

```diff
@@ -1,39 +1,65 @@
 <!DOCTYPE html>
 <html>
-	<head>
-		<meta charset="UTF-8">
-		<title>My Portfolio</title>
-		<link rel="stylesheet" href="style.css">
-		<script src="script.js"></script>
-	</head>
-	<body>
-		<div id="content">
-			<h1>Jenny's Portfolio</h1>
-
-			<table>
-				<th onclick="Toggle('main-section')">home</th>
-				<th onclick="Toggle('about-section')">about me</th>
-				<th onclick="Toggle('contact-section')">contact me</th>
-			</table>
-
-				
-			<div id="main-section">
-				<img id="headshot" src="/images/Headshot_Jenny.JPG" alt="Image of me."/>
-				<p>Click here to see a random fact about me!:</p>
-				<button onclick="addRandomGreeting()">Hello</button>
-				<div id="greeting-container"></div>
+
+<head>
```

Comment:
The indentation seems weird. Can you run the formatter again?

Reply:
Hm, I'm not sure why it shows up on GitHub like this. In the CloudShell editor, the code seems to be correctly indented (using two spaces). I tried fixing it, and ran git diff, but there is no difference.